tests/dragonfly/utility.py

```diff
@@ -233,7 +233,7 @@ def gen_update_cmd(self):
         ValueType.SET: "SADD",
         ValueType.HSET: "HMSET",
         ValueType.ZSET: "ZADD",
-        ValueType.JSON: "JSON.SET",
+        ValueType.JSON: "JSON.MSET",
```

Comment:
why replace? 🤔

Reply:
Because it's not trivial to duplicate those. Why wouldn't MSET cover SET fully? I think it's ok